### PR TITLE
ci: verify our boilerplate

### PR DESCRIPTION
**Pull request by stevekuznetsov**
This is a non-standard invocation as we don't want to pull in Bazel
right now.

Signed-off-by: Steve Kuznetsov <skuznets@redhat.com>

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,3 +1,17 @@
+# Copyright 2021 The Kubernetes Authors.
```
Comment (imjasonh) [inline Makefile:1]: Ianal, is this what we want here? `The KCP Authors`?

**Comment by imjasonh** [comment]
Another option that Knative uses: https://github.com/mattmoor/boilerplate-check

**Comment by stevekuznetsov** [comment]
I think this needs? https://github.com/kubernetes/repo-infra/pull/246

File: Makefile (hunk 1 above)
Comment (stevekuznetsov) [reply Makefile:1]: /shrug

Open to ideas, no clue

**Comment by smarterclayton** [comment]
generally rh legal allows license file at the top for apache projects and no boilerplate

**Comment by ncdc** [comment]
That would certainly make things easier (less process). Does that work for you @stevekuznetsov (I know I was the one yesterday asking about missing boilerplate in one of your PRs, but ... 😄 )

**Comment by smarterclayton** [comment]
but!  we want to remain generally aligned to kube specifically and cncf convention generally - to continue with existing codebase as a kube sub project will require us to establish ownership and some of what we’re doing might have to get rewritten.  i think it’s likely anything minimal api server “like” needs to be very much associated with the kube fork (at the point we suggest merges based on that is when ownership is transferred to kube and cncf)

**Review by imjasonh — APPROVED** [review]

**Pull request merged**